frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/details-card.tsx

```diff
@@ -117,6 +176,28 @@ const DetailsCard: React.FC<DashboardItemProps> = ({
             error={subscriptionLoaded && !ocsVersion}
           >
             {ocsVersion}
+            {!_.isEmpty(filteredVersions) &&
+              memoizedOcsSubscription?.spec?.installPlanApproval === 'Manual' && (
+                <>
+                  {ocsSubscriptionLoaded && packageManifestLoaded && (
+                    <Button
+                      type="button"
+                      isInline
+                      onClick={launchModal}
+                      variant="link"
+                      className="pf-u-ml-xl"
+                    >
+                      <BlueArrowCircleUpIcon className="co-icon-space-r" />
+                      {t('ceph-storage-plugin~Update to ')}
+                      {
+                        memoizedPackageManifest.status.channels[
+                          memoizedPackageManifest.status.channels.length - 1
+                        ].name
```

Comment:
Take it out in a variable. Also is the order always there?

Reply:
Yes. I see, it's in ascending order.